usage-tests/nodejs/src/index.js

```diff
@@ -0,0 +1,4 @@
+const { howMuch } = require("howmuchcarbon");
+const fs = require("fs");
+
+console.log(howMuch("hello"));
```

Comment:
```suggestion
console.log(howMuch("iPad"));
```

Maybe, so it looks more realistic?